src/components/Button/styledicon.js

```diff
@@ -1,6 +1,9 @@
 import styled from 'styled-components';
 
 const StyledIcon = styled.button`
+	position: ${props => (props.modalIcon ? 'absolute' : '')};
+	top: ${props => (props.modalIcon ? '13em' : '')};
+	left: ${props => (props.modalIcon ? '16em' : '')};
```

Comment:
Another way to write this would be with the `css` functionality from styled-components.
```suggestion
    ${props => props.modalIcon && css`
          position: absolute;
          top: 13em;
          left: 16em;
        `
    }
```
The `css` function of styled-components allows you to add multiple lines of CSS styles within another template string.
Here is an example of how to use this as well: https://codesandbox.io/s/button-variants-zwhy6u?file=/src/Button.js